system-tests/default-excludes-test/src/test/java/com/teamscale/tia/client/TeamscaleMockServer.java

```diff
@@ -30,7 +30,7 @@
 
 	public TeamscaleMockServer(int port) throws IOException {
 		port(port);
-		post("api/projects/:projectName/external-analysis/session/auto-create/report", this::handleReport);
+		post("api/v5.9.0/projects/:projectName/external-analysis/session/auto-create/report", this::handleReport);
```

Comment:
Why are all these methods calling the APIs of specific Teamscale versions?

Reply:
Because otherwise we will break the agent once we introduce a breaking change to the API in the future. Instead when all external tools specify the version they expect (like the Java based IDEs already do) we can safely change stuff in the future without having to way years until all tools and instances are updated (as we previously had to)

Reply:
I mean isn't that the whole point of versioned APIs